src/schema/schema.ts

```diff
@@ -124,7 +31,9 @@ class Schema {
    *  const schema = new Schema([new StringType('name')]);
    * ```
    */
-  constructor(private fields: CoreType[]) {}
+  constructor(private fields: CoreType[]) {
+    this._id = '_id';
+  }
   /**
    * Validate an model instance using definition of schema
```

Comment:
Should be: `Validate (a) model instance using definition of schema`

Reply:
Fixed: Validate a model instance using definition of schema